store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/integration/FederatedStoreRecursionIT.java

```diff
@@ -75,7 +75,7 @@ public void setUp() throws Exception {
     }
 
 
-    @Test(timeout = 60000)
+    @Test//(timeout = 60000)
```

Comment:
Can this be removed?

Reply:
Good spot. I added that for debugging. My bad.